tests/fs/test_mmap.c

```diff
@@ -176,5 +176,53 @@ int main() {
     printf("/yolo/sharedoffset.txt content=%s %d\n", buffer + offset, offset);
     fclose(fd);
   }
+
+  /**
+   * MMAP to and expanded file
+   * 
+   * When appending to a file, the buffer is expanded in chunks, and so the actual length
+   * of the file could be less than the buffer length.
+   * 
+   * When using mmap for an expanded file, we have to make sure that content from the buffer
+   * is not written beyond the allocated memory area for the mmap operation.
+   */
+  {
+    const char* path = "/yolo/expandedfile.txt";
+
+    int fd = open(path, O_RDWR | O_CREAT, (mode_t)0600);
+    assert(fd != -1);
+
+    size_t textsize = 33;
+
+    // multiple calls to write so that the file is expanded
+    for(int n=0;n<textsize;n++) {
```

Comment:
space after `for` and between all the operators.

Reply:
looking better now?